packages/block-editor/src/components/colors-gradients/style.scss

```
@@ -120,3 +121,17 @@ $swatch-gap: 12px;
 		flex-shrink: 0;
 	}
 }
+
+.block-editor-panel-color-gradient-settings__reset {
+	position: absolute;
+	right: 0;
+	top: 0;
+	opacity: 0;
+	transition: opacity 0.1s ease-in-out;
```

```suggestion
	transition: opacity 0.1s ease-in-out;
	@include reduce-motion("transition");
```

If we use the `transition`, we need to provide users with a way to opt out of them, whenever possible.